### PR TITLE
Add InsightsLabel component to pdf-generator

### PR DESCRIPTION
**Pull request by LI0131**
This PR is in response to https://projects.engineering.redhat.com/browse/RHCLOUD-8451.

It adds the `InsightsLabel` component to the `pdf-generator` package. This component is a recreation of the component of the same name found in the `components` package. The addition of this component will allow for us to move away from the usage of the Battery component completely.

Here are some screenshots of the new svgs:

Screen Shot 2020-08-07 at 1 40 37 PM

Screen Shot 2020-08-07 at 1 47 55 PM

### CODE REVIEW

File: packages/pdf-generator/src/components/InsightsLabel.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, Canvas, Text } from '@react-pdf/renderer';
+import styles from '../utils/styles';
+
+const appliedStyles = styles();
+
+//eslint-disable-next-line
```
Comment (AllenBW) [inline packages/pdf-generator/src/components/InsightsLabel.js:9]: Leeeeeetsssss just do this once at  at the top of the file

/* eslint max-len: 0 */

File: packages/pdf-generator/src/utils/styles.js [hunk 2]
```diff
@@ -94,6 +104,26 @@ export default (style = {}) => StyleSheet.create({
...
+        textColor: global_palette_blue_600.value,
+        iconColor: global_primary_color_100.value
+    },
+    labelColorsModerate: {
+        bgColor: global_palette_gold_50.value,
+        textColor: global_palette_gold_700.value,
+        iconColor: chart_global_warning_Color_200.value
+    },
+    labelColorsImportant: {
+        bgColor: '#fff5ec',
```
Comment (AllenBW) [inline packages/pdf-generator/src/utils/styles.js:118]: does this not have a pf color? 🤔

Comment (LI0131) [reply packages/pdf-generator/src/utils/styles.js:118]: I believe that's a custom color. It's the same in the https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/Components/InsightsLabel/labels.scss

File: packages/pdf-generator/src/components/InsightsLabel.js (hunk 1 above)
Comment (LI0131) [reply packages/pdf-generator/src/components/InsightsLabel.js:9]: 💯 🥇

**Comment by LI0131** [comment]
@AllenBW someone can correct me if I'm mistaken here, but I'm pretty sure that since react-pdf is using it's own renderer you have to pass components in which can be rendered by it. We do the same thing with the Battery component currently.

File: packages/pdf-generator/src/components/InsightsLabel.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    4: {
+        text: 'Critical',
+        iconPath: criticalPath,
+        width: 55,
+        ...appliedStyles.labelColorsCrit
+    }
+};
+
+const InsightsLabel = ({ variant }) => {
```
Comment (karelhala) [inline packages/pdf-generator/src/components/InsightsLabel.js:42]: You should take in the rest of the props, in order to allow `debug` option or custom styling.
```suggestion
const InsightsLabel = ({ variant, ...props }) => {
```

File: packages/pdf-generator/src/components/InsightsLabel.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    const { bgColor, iconColor, iconPath, width, text, textColor } = labelMapper[variant];
+    return <View style={appliedStyles.flexRow}>
```
Comment (karelhala) [inline packages/pdf-generator/src/components/InsightsLabel.js:44]: Here you should spread those props
```suggestion
    return <View style={appliedStyles.flexRow} {...props}>
```

**Comment by terezanovotna** [comment]
Very nice job!! our reports look great!!

**Pull request merged**